src/index.ts

```diff
@@ -289,48 +344,84 @@ const ex = express()
 		latestLogs.splice(0, latestLogs.length - maxY)
 	})
 
-	next()
+	return res
+}
+
+//? #################################
+//? ############# SERVER ############
+//? #################################
+
+const exStatic = serv('./', {
+	dotfiles: DOTFILES,
+	etag: ETAG,
+	extensions: EXTENSIONS,
+	index: INDEX,
+	maxAge: MAX_AGE,
+	logger: (req, res) => reqLog(req, res as Response),
 })
 
-.use('/', (req, res, next) => { //? General handler
-	let path = decodeURI(req.path)
-	if (req.path.startsWith('//')) {
+const genHandler: RequestListener = (req, res) => { //? General handler
+	const reqPath = getPath(req.url!)
+	let path = decodeURI(reqPath)
+	if (req.method !== 'GET') return notFoundHandler(req, res)
+	if (reqPath.startsWith('//')) {
 		res.locals.fs = true
 		path = path.slice(1)
-		return next()
+		return fileHandler(req, res)
 	}
 
 	let [redir, newPath] = route(encodeURI(path))
-	req.url = newPath + req.originalUrl.split('?').splice(1).join('?')
+	req.url = newPath + req.url!.split('?').splice(1).join('?')
 
-	if (redir) return res.redirect(req.url)
+	if (redir) {
+		res.log()
+		return redirect(res, req.url)
+	}
```

Comment:
After creating `res.redirect()`, this can be simplified.
```suggestion
	if (redir) return res.log().redirect(req.url)
```